### ISSUE TITLE
evaluation code

### ISSUE THREAD

**Issue by orgw** [aidd-msca/GraphQPT#1]
Hi thanks for the code,

is there a code and pretrained model for evaluation? I would like to see the results of the paper. It would be much appreciated

**Comment by JacksonBurns** [comment]
+1 for pretrained model weights being made available

**Comment by shulaibao123-i** [comment]
Yes, it would be great if you could provide the pretrained model. Thank you very much!
